gen/scim

```diff
@@ -70,16 +70,16 @@ foreach my $uid (sort keys %$userStruc) {
 # PRINT USERS TO JSON
 open FILE_USERS,">$fileUsers" or die "Cannot open $fileUsers: $! \n";
 binmode(FILE_USERS);
-print FILE_USERS JSON::XS->new->utf8->pretty->encode(\@users);
+print FILE_USERS JSON::XS->new->utf8->pretty->canonical->encode(\@users);
 close (FILE_USERS) or die "Cannot close $fileUsers: $! \n";
 
 # PREPARE GROUPSDATA TO JSON
 my @groups;
 foreach my $gid (sort keys %$groupStruc) {
         my $group = {};
         $group->{"id"} = $gid;
-        $group->{"name"} = $groupStruc->{$gid}->{$A_GROUP_NAME};
-        $group->{"parentGroupId"} = $groupStruc->{$gid}->{$A_GROUP_PAR_ID};
+        $group->{"name"} = $data->getGroupAttributeValue(group => $gid, attrName => $A_GROUP_NAME);
+        $group->{"parentGroupId"} = $data->getGroupAttributeValue(group => $gid, attrName => $A_GROUP_PAR_ID);
```

Comment:
I think you can get the attributes from the `groupStruc` just like it used to do or you don't have save it in there during `processGroups` method.

Reply:
You're right, kept the old approach.